mintpy/objects/gps.py

```diff
@@ -168,11 +169,12 @@ def get_gps_los_obs(insar_file, site_names, start_date, end_date,
             # calculate gps data value
             obj = GPS(site_name)
 
-            if obs_type == 'velocity':
+            if obs_type == 'velocity' or obs_type == 'displacement':
```

Comment:
`displacement` should be together with `timeseries`.

Reply:
Or how about we calculate/save the velocity all the time by getting rid of the `if elif` and use `obj.get_gps_los_velocity()` only? We would need to adjust `site_obs` line below for the obs_type check.